src/test/end-to-end-tests/test_suites/play_test.ts

```diff
@@ -0,0 +1,228 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-console */
+import { KmqResponseType } from "./test_suite";
+import type * as Eris from "eris";
+import type ParsedGameOptionValues from "../parsed_game_options_value";
+import type TestSuite from "./test_suite";
+
+// lightweight check to see that bot is still alive and responding to messages
+const PLAY_TEST_SUITE: TestSuite = {
+    name: "Healthcheck",
+    tests: [
+        {
+            command: ",reset",
+            responseValidator: (
+                title: string,
+                description: string,
+                parsedGameOptions?: ParsedGameOptionValues,
+                client?: Eris.Client,
+            ) => title === "Options",
+            expectedResponseType: KmqResponseType.GAME_OPTIONS_RESPONSE,
+            requiresVoiceConnection: true,
+        },
+        {
+            command: ",limit 6",
+            responseValidator: (
+                title: string,
+                description: string,
+                parsedGameOptions?: ParsedGameOptionValues,
+                client?: Eris.Client,
+            ) =>
+                parsedGameOptions!["limit set top"]!.value.startsWith("6 /") &&
+                parsedGameOptions!["limit set top"]!.updated,
+            expectedResponseType: KmqResponseType.GAME_OPTIONS_RESPONSE,
+            requiresVoiceConnection: true,
+        },
+        {
+            command: ",shuffle popularity", // deterministic ordering so we can figure out how to guess
```

Comment:
Will this ordering break once one of these six songs changes rank? Should we query the DB to get the song/artist names instead

Reply:
Yes but would rather keep the test runner agnostic of anything internal, it should be as close to a regular player as possible. 

Many of our tests break when the database changes in certain ways. We use a cached version of the database so it doesn't break unless we update it, and fixing things doesn't take much effort

Reply:
Good point, let's leave it as is then